brandon-vagarioustoast/week-3/giffaw/scripts/keys.js

```diff
@@ -0,0 +1,2 @@
+const apiKey = 'hbanKxYFVAFdYoBZPEKihtN7jfR8fjFS';
```

Comment:
Good job putting your key into a separate file, but in this sort of application, this will do nothing to hide your key as it will still be visible. Also, a key was provided in the HTML input. (Sneaky I know) This is why it is always important to read all of the provided code ahead of coding to save on development time. This is a practice that will be important in a professional work environment.